tests/Mappers/Parameters/TypeMapperTest.php

```diff
@@ -77,6 +101,10 @@ private function dummy() {
 
     }
 
+    private function objectUnion(): TestObject|TestObject2 {
```

Comment:
I also tried `null|TestObject|TestObject2` but that failed, not sure if the library needs to support that